osf/migrations/0043_add_quickfiles.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11 on 2017-06-13 17:35
+from __future__ import unicode_literals
+
+import logging
+
+from django.db import migrations, models
+from django.core.paginator import Paginator
+
+from osf.models import OSFUser, QuickFiles, Contributor
+from osf.models.quickfiles import get_quickfiles_project_title
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+def add_quickfiles(*args, **kwargs):
+    ids_without_quickfiles = list(OSFUser.objects.exclude(created__type='osf.quickfiles').values_list('id', flat=True))
```

Comment:
Yes! They will be the exact same! I originally had exactly what you suggest here, but ran into a lot of trouble when re-writing the migration to use pagination. It turns out that for pagination, the query was re-run every time, leading to a weird moving target issue where `users_without_quickfiles` would shrink every page, and therefore never complete all available pages.

I went over this with @sloria for quite awhile, and this was the solution (although definitely weird and verbose) solution we came up with.  I tried running it several different times with different methods (I have the data for using timeit if you wanna see) and this was the fastest that actually was able to get through all the pages.